pyteal/ast/itxn.py

```diff
@@ -200,6 +206,111 @@ def SetFields(cls, fields: Dict[TxnField, Union[Expr, List[Expr]]]) -> Expr:
         fieldsToSet = [cls.SetField(field, value) for field, value in fields.items()]
         return Seq(fieldsToSet)
 
+    @classmethod
+    def MethodCall(
+        cls,
+        *,
+        app_id: Expr,
```

Comment:
Are any of these arguments mandatory? It ought to be the case that `app_id` is since how else would you know which app to call and re-entry is disallowed.

Reply:
NVM - this is already being enforced